### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by TalReinfeld**
Profile sync to the system theme.

Users are required to login to the site.

URL for a logged in user is http://127.0.0.1:8000/profile/ - no matter if it is a Client or a Professional.

Connect the features to the profile page.

Client:

image

image

image

image

Professional:

image

image

image

### CODE REVIEW

**Review by idosinger98 — COMMENTED** [review]
please convert to draft

**Review by patrisiakaplun — COMMENTED** [review]
There are a lot of commits here, so it is very difficult to follow the chain of events
Can you squash it a bit?

**Review by Yarboa — COMMENTED** [review]
Please open a ticket to remove the 
demo_client, demo_client2 and proffessional there are new factories for that

You can remove the static init of the user password, you can set the data from test per demand and assert the content in the tests

File: account/tests/urls_tests/test_general.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import pytest
+
+
+PROFILE_URL = '/profile/'
+
+
+@pytest.mark.django_db
+class TestGeneral:
+    def test_get_not_registered_entity(self, client):
```
[the remark this answers is not in the document]
Comment (TalReinfeld) [reply account/tests/urls_tests/test_general.py]: The test check if a not registered user is trying to use a URL that allowed only for registered users.
In that case he will be redirected to the landing page.
Therefore the new name of the test is - test_redirection_of_not_registered_entity.
@Yarboa

**Comment by TalReinfeld** [comment]
> Please open a ticket to remove the demo_client, demo_client2 and proffessional there are new factories for that
> 
> You can remove the static init of the user password, you can set the data from test per demand and assert the content in the tests

A ticket was opened, issue #138 

@Yarboa

**Review by idosinger98 — APPROVED** [review]
LGTM

**Review by guybeckenstein — APPROVED** [review]
LGTM

File: account/templates/account/profile.html [hunk 2]
```diff
@@ -1,70 +1,74 @@
...
+                            <li><i class="fas fa-briefcase"></i><div><div class="litext">
+                            <a href="#">
+                                Type Of Jobs
+                            </a>
+                            </div></div></li>
+                            <li><i class="far fa-clock"></i><div><div class="litext">
+                            <a href="{% url 'calendar'%}">
+                                Schedule
+                            </a>
+                            </div></div></li>
```
[the remark this answers is not in the document]
Comment (TalReinfeld) [reply account/templates/account/profile.html:48]: Done @guybeckenstein

**Review by idosinger98 — COMMENTED** [review]
You have some issues with the design

File: account/templates/account/login.html [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
+    <link rel="stylesheet" href="{% static 'css/base_style.css' %}">
+{% endblock %}
 <!--Add body content dynamically-->
 {%block content%}
-  <div class="center login-center" style="margin-top: -50px;">
+  <div class="center login-center">
         <!--Login input-->
-	  <form method="post" class="card" novalidate style="">
+	  <form method="post" class="card" novalidate style="margin-top: 100px">
 	  	{% csrf_token %}
```
Comment (idosinger98) [inline account/templates/account/login.html:17]: Please do not modify this file

**Review by idosinger98 — APPROVED** [review]
LGTM

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM
Although I think a ticket should be opened to deal with the linked mechanism 
You should avoid the [X] usage

https://github.com/redhat-beyond/Pro4U/pull/127/files#diff-a31c7ed5d35f5ed8233994868c54d625b18e6bacb6794344c4531e62bd9dde59R26-R43

**Comment by TalReinfeld** [comment]
@Yarboa 
Can you explain what do you mean when you say "You should avoid the [X] usage"?

**Comment by Yarboa** [comment]
> @Yarboa Can you explain what do you mean when you say "You should avoid the [X] usage"?

Avoid call like those now 
https://github.com/redhat-beyond/Pro4U/pull/127#issuecomment-1576659018

**Pull request merged**